cert-manager/overlays/moc/smaug/api/openshift-api-issuer.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: ingress-letsencrypt
```

Comment:
Because this issuer isn't for the `openshift-ingress` namespace we should name it something other than `ingress-letsencrypt`. Perhaps `api-letsencrypt`? You'll need to update the corresponding information in the `Certificate` resource.